assignments1/secondlargestArray.js

```diff
@@ -0,0 +1,20 @@
+function secondLargest(array) {
+ 
+  for(var iter=0;iter<array.length;iter++)
+    {
+        for(var sec_iter=0;sec_iter<array.length-iter-1;sec_iter++)
```

Comment:
variables are supposed to be named in **camelCasing** and not **underscore_separated**